cmake/makefile_exports/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+
+set(exports
+  "
+CC = ${CMAKE_C_COMPILER}
+CXX = ${CMAKE_CXX_COMPILER}
+OBJCOPY = ${CMAKE_OBJCOPY}
+OBJDUMP = ${CMAKE_OBJDUMP}
+AS = ${CMAKE_AS}
+AR = ${CMAKE_AR}
+NM = ${CMAKE_NM}
+GDB = ${CMAKE_GDB}
+KBUILD_CFLAGS = \"-I$<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_INCLUDE_DIRECTORIES>, -I> -isystem $<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_SYSTEM_INCLUDE_DIRECTORIES>, -isystem > -D$<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_COMPILE_DEFINITIONS>, -D> $<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_COMPILE_OPTIONS>, >\"
```

Comment:
2 points here:

1. While long ago this var indeed might have been named KBUILD_CFLAGS, I guess naming it like that now would only lead to confusion, there's no Kbuild any longer. In my recent workarounds, I named it Z_CFLAGS, and I propose to do just that. (But it must have a prefix, yes. The typical usage pattern in 3rd-party Makefile is: `CFLAGS = -Iapp/includes $(Z_CFLAGS)`).

2. Crucially, this value should not be quoted. As soon as I remove quotes manually from generated Makefile.exports.C, by build works. With quotes, it's `mpconfigport.h:31:10: fatal error: zephyr.h: No such file or directory`.

Reply:
Just confirming that removing `\"` here solves the issue. (That would be obvious, but sometimes quote quoting issue can get quite nasty...)

Reply:
Renamed to Z_CFLAGS.

Removed quotes.